homeassistant/components/konnected/config_flow.py

```diff
@@ -205,6 +209,28 @@ def __init__(self):
 
         # if we have no host info wait for it or abort if previously configured
         self._abort_if_unique_id_configured()
+        return await self.async_step_import_confirm()
+
+    async def async_step_import_confirm(self, user_input=None):
+        """Confirm the user wants to import the config entry."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="import_confirm",
+                description_placeholders={"id": self.unique_id},
+            )
+
+        # if we have ssdp discovered applicable host info use it
```

Comment:
Do we really need this? If the user wants discovery help the user can use that approach instead of config yaml.

Reply:
After receiving feedback on the beta build I think we do need to support this use case.  The legacy konnected component automatically discovered host info and then applied the config.yaml.  

The config.yaml can be extensive for this integration so we shouldn't require users to abandon what they have.  This flow makes for a smooth upgrade path existing konnected users.

Reply:
Ok.